requirements/common.in

```diff
@@ -144,7 +144,7 @@ py3dns
 
 # Install Python Social Auth
 social-auth-app-django
-social-auth-core[azuread,saml]
+https://github.com/python-social-auth/social-core/archive/3.4.0.zip/#egg=social-auth-core[azuread,saml]==3.4.0
```

Comment:
Have you asked upstream why this hasn’t been released to PyPI?

Reply:
No, just opened an issue asking. https://github.com/python-social-auth/social-core/issues/485